src/ResourceManager/Websites/Commands.Websites/ChangeLog.md

```diff
@@ -18,12 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+* New-AzureRmAppServicePlan -HyperV switch is added for create app service plan with windows container
+* New-AzureRmWebApp/ New-AzureRmWebAppSlot/ Set-AzureRmWebApp/ Set-AzureRmWebAppSlot - New parameters (–ContainerRegistryUser string -ContainerRegistryPassword secureString -EnableContainerContinuousDeployment) added for creating and managing windows container app
+* New Cmdlets Enter-AzureRmWebAppContainerPSSession, New-AzureRmWebAppContainerPSSession for entering and creating remote PSSession in container web app
+* New Cmdlet Get-AzureRMWebAppContainerContinuousDeploymentUrl - Gets the Container Continuous Deployment Webhook URL
 
 ## Version 5.1.0
 * Updating to use the latest .NET SDK version (2.0.0)
 * Added two new cmdlets: Get-AzureRmDeletedWebApp and Restore-AzureRmDeletedWebApp
-* New-AzureRmAppServicePlan -HyperV switch is added for create app service plan with windows container
-* New-AzureRmWebApp/ New-AzureRmWebAppSlot/ Set-AzureRmWebApp/ Set-AzureRmWebAppSlot - New parameters (�ContainerRegistryUser string -ContainerRegistryPassword secureString -EnableContainerContinuousDeployment) added for creating and managing windows container app
```

Comment:
Keep these 2 lines here. Just have the other 2 lines in the `Current Release` header.